src/components/diet-restrict/DietRestrict.js

```diff
@@ -13,19 +13,38 @@ export class DietRestrict extends Component {
         return (
             <div>
                 <p> dietary restriction</p>
-                <form >
-                <input 
-                    list="veto" 
-                    placeholder = "cuisineVeto"
-                    onChange = {this.props.handleChange('dietRest')}
-                />
-                    <datalist id="veto">
-                        <option value="1">Chinese</option>
-                        <option value="2">Western</option>
-                        <option value="3">Japanese</option>
-                        <option value="4">Korean</option>
-                    </datalist>
-                </form>
+                <label>
+                    Chinese
+                    <input 
+                    name = "restChinese"
+                    type = "checkbox" 
+                    checked = {this.props.restChinese} 
+                    onChange = {this.props.handleInputChange} />
+                </label>
```

Comment:
somehow this feels like a pretty convoluted way to do this. hmm. could you possibly handle the state for these restaurants internally and only return it after the user hits next?

along these lines - might need a tweak of `handleInputChange` for this to work
```
class DietRestrict extends Component {
    state = {chinese:0, korean:0, ...}

    handleCheck = (cuisine) => {
        this.setState({
            ...this.state,
            [cuisine]: !this.state[cuisine]
        })
    }

    nextStep = () => {
        this.props.handleInputChange(this.state)
        this.props.nextStep()
    }

    render() {
        <>
            <label>
                Japanese
                <input 
                    name = "restJapanese"
                    type = "checkbox" 
                    onChange = {this.handleCheck('japanese')} />
            </label>
            <NextButton nextStep={nextStep}/>
        </>
    }
  }
```